src/HomeKeyPlugIns.ZC

```diff
@@ -110,6 +110,16 @@ Bool MyPutKey(I64 ch, I64 sc)
 						}
 						return TRUE;
 
+					case SC_F12:
+						if (!(sc & SCF_SHIFT))
+						{
+							if (sc & SCF_KEY_DESC)
+								KeyDescSet("Cmd /Yuugure");
+							else
+								Yuugure;
```

Comment:
This has the same issue as the other PR, which forces the script to be loaded prior to referencing it.
Is there no way to counter that?